extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -404,7 +404,6 @@ private void handleHibernateORMWithNoPersistenceXml(
                 // sql-load-script
                 // explicit file or default one
                 String file = hibernateConfig.sqlLoadScript.orElse("import.sql"); //default Hibernate ORM file imported
-
```

Comment:
could you remove this change? Not every important but it seems unrelated with the PR :)

Reply:
Good catch.  A mistake on my end. :-)